Weather/ViewModels/CityViewModel.swift

```diff
@@ -11,6 +11,7 @@ import CoreLocation
 
 class CityViewModel: ObservableObject {
     
+    @Published var loading = false
     @Published var weather = WeatherResponse.emptyResponse()
```

Comment:
Why do you still use `WeatherResponse.emptyResponse()` instead of nil? This way you won't need to create a separate variable `loading`